media_driver/agnostic/common/vp/hal/vphal.cpp

```diff
@@ -789,11 +789,7 @@ MOS_STATUS VphalState::GetStatusReport(
             continue;
         }
 
-#if LINUX
```

Comment:
Please keep the macro. If you want to enable it on Android, you can change it to:
#if (LINUX || ANDROID)